Training_Center_Task_10/Tests.cs

```diff
@@ -0,0 +1,301 @@
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+
+            //Click "Click me!" button
+            IWebElement ClickMeButton = _driver.FindElement(By.CssSelector("button[onclick='myConfirmFunction()']"));
+            ClickMeButton.Click();
+
+            _driver.Wait(2000);
+
+            //Cliok OK on alert
+            _driver.SwitchTo().Alert().Accept();
+
+            IWebElement alertClickedOk = _driver.FindElement(By.Id("confirm-demo"), 5);
+
+            Assert.AreEqual("You pressed OK!", alertClickedOk.Text);
+        }
+
+        [Description("Task 6")]
+        [TestMethod]
+        public void AlertTestJavaScriptAlertBoxNameEnteredCorrectly()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+
+            //Click "Click for Prompt Box" button
+            IWebElement ClickForPromptBoxButton = _driver.FindElement(By.CssSelector("button[onclick='myPromptFunction()']"));
+            ClickForPromptBoxButton.Click();
+
+            _driver.Wait(2000);
+
+            //Enter the name in alert and click OK
+            _driver.SwitchTo().Alert().SendKeys("Ivo");
+            _driver.SwitchTo().Alert().Accept();
+
+            IWebElement message = _driver.FindElement(By.Id("prompt-demo"), 5);
+
+            Assert.AreEqual("You have entered 'Ivo' !", message.Text);
+        }
+
+        [Description("Task 7")]
+        [TestMethod]
+        public void WaitForUser()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/dynamic-data-loading-demo.html");
+
+            //Click "Get New User" button
+            IWebElement getNewUserButton = _driver.FindElement(By.Id("save"), 5);
+            getNewUserButton.Click();
+
+            //Verify "loading..." message is displayed
+            IWebElement loadingMessage = _driver.FindElement(By.Id("loading"));
+            Assert.AreEqual(true, loadingMessage.Displayed);
+
+            //Verify that after 3 seconds of waiting a new user is displayed
+            _driver.Wait(3000);
+            Assert.AreEqual(true, getNewUserButton.Displayed);
+        }
+
+        [Description("Task 8")]
+        [TestMethod]
+        public void RefreshWhenDownloadSizeBiggerThan50()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/bootstrap-download-progress-demo.html");
+
+            //Click "Download" button
+            IWebElement downloadButton = _driver.FindElement(By.Id("cricle-btn"), 5);
+            downloadButton.Click();
+            _driver.Wait(2000);
+
+            //Track download progress
+            DefaultWait<IWebDriver> fluentWait = new DefaultWait<IWebDriver>(_driver);
+            fluentWait.Timeout = TimeSpan.FromSeconds(5);
+            fluentWait.PollingInterval = TimeSpan.FromMilliseconds(250);
+            fluentWait.IgnoreExceptionTypes(typeof(NoSuchElementException));
```

Comment:
not a good way to ignore such exceptions

Reply:
Changed to Exception